mpas_analysis/ocean/time_series_ohc.py

```diff
@@ -292,43 +324,244 @@ def run(self):  # {{{
         for regionIndex in regionIndicesToPlot:
             region = regions[regionIndex]
 
-            ohc = dsOHC.ohc.isel(nOceanRegionsTmp=regionIndex)
+            # Plot temperature, salinity and OHC anomalies (and full fields)
+            # trends with depth
+
+            #  First T vs depth/time
```

Comment:
In the long run, I think it would be good to get some better code reuse out of this section of code.  My thought would be to do this a bit more like I've done for climatology maps where you have a base class that plots Hovmoller diagrams and/or anomaly time series of whatever you want.  Then, you have several subclasses that are for doing such plots with T, S, OHC or whatever you like.  The subclasses define the specific things like axis labels, titles, file prefixes, etc.

I think this is too much for this PR, but be aware that I will likely make this change sometime in the future so it's easier to add new fields.